NAMESPACE

```diff
@@ -2,9 +2,9 @@ S3method("+",Foo)
 S3method(hello,character)
 S3method(hello,default)
 export(hello)
-export(reverse)
+export(reverse, reverse_int)
 export(rock_fun)
 export(split_filename)
 importFrom(tools,file_ext)
 importFrom(tools,file_path_sans_ext)
-useDynLib(rmini)
+useDynLib(rmini, .registration = TRUE, .fixes = "rmini_")
```

Comment:
Are the two lines of changes generated by roxygen2 or by hand? I'm asking because I don't see how they could be generated by roxygen2 (e.g., you didn't have `@export` for `reverse_int()` in `R/C.R`).

Reply:
by hand! didn't realize roxygen was enabled for the package. I'll use document for this instead...

Reply:
you know I have no idea how this was added on second glance.